deps/build.jl

```diff
@@ -43,3 +44,60 @@ end
 compile(TZDATA_DIR, COMPILED_DIR)
 
 info("Successfully processed TimeZone data")
+
+
+isdir(WINDOWS_DIR) || mkdir(WINDOWS_DIR)
+
+# Windows is weird and uses its own timezone
+info("Downloading Windows to TZ data")
+
+# Generate the mapping between MS Windows timezone names and
+# tzdata/Olsen timezone names, by retrieving a file.
+if 1==1
+    xml_source = "http://unicode.org/cldr/data/common/supplemental/windowsZones.xml"
+    xml_file = joinpath(WINDOWS_DIR, "windowsZones.xml")
```

Comment:
Maybe just have this in the `deps` folder? Why does this need a brand new directory?